book/src/pod-info.md

```diff
@@ -17,21 +17,18 @@ Specifically the `podInfoOnMountVersion` field instructs Kubernetes that the CSI
 For example, the existence of the following object would cause Kubernetes to add pod information at mount time to the `NodePublishVolumeRequest.volume_context` map.
 
 ```
-apiVersion: csi.storage.k8s.io/v1alpha1
+apiVersion: storage.k8s.io/v1beta1
 kind: CSIDriver
 metadata:
   name: testcsidriver.example.com
 spec:
-  podInfoOnMountVersion: v1
+  podInfoOnMountVersion: true
 ```
 
-There is only one `podInfoOnMountVersion` version currently supported: `v1`.
-
-The value `v1` for `podInfoOnMountVersion` will result in the following key/values being added to `publish_context`:
+`true` value in `podInfoOnMountVersion` will result in the following key/values being added to `publish_context`:
 
 * `csi.storage.k8s.io/pod.name: {pod.Name}`
 * `csi.storage.k8s.io/pod.namespace: {pod.Namespace}`
 * `csi.storage.k8s.io/pod.uid: {pod.UID}`
 
-
 The easiest way to use this feature is to deploy the [cluster-driver-registrar](cluster-driver-registrar.md) sidecar container. Once the flags to this container are configured correctly, it will automatically create a [CSIDriver Object](csi-driver-object.md) when it starts with the correct fields set.
```

Comment:
## Alpha Functionality

In alpha, this feature was enabled by setting the `podInfoOnMountVersion` field in the [CSIDriver Object](csi-driver-object.md) CRD to to `v1`.

```
apiVersion: csi.storage.k8s.io/v1alpha1	
kind: CSIDriver
metadata:
  name: testcsidriver.example.com
spec:
  podInfoOnMountVersion: v1
```